web/packages/teleterm/src/ui/TopBar/Identity/IdentityList/IdentityList.tsx

```diff
@@ -66,9 +60,7 @@ export function IdentityList(props: IdentityListProps) {
             <IdentityListItem
               key={i.uri}
               index={index}
-              isSelected={i.active}
-              userName={i.userName}
-              clusterName={i.clusterName}
+              cluster={i}
```

Comment:
dont need to change because it already exists, but `i` for the cluster and then `index` for the index is just... idk